### ISSUE TITLE
Latest releases miss 7 month old change(s) from master branch

### ISSUE THREAD

**Issue by dtugend** [woocommerce/woocommerce#20657]
Example:

```
$attributes[] = apply_filters( 'woocommerce_admin_meta_boxes_prepare_attribute', $attribute, $data, $i );
```
https://github.com/woocommerce/woocommerce/blame/master/includes/admin/meta-boxes/class-wc-meta-box-product-data.php#L275

vs

```
$attributes[] = $attribute;
```
https://github.com/woocommerce/woocommerce/blame/3.4.3/includes/admin/meta-boxes/class-wc-meta-box-product-data.php#L275

Something is wrong with your merging maybe :(

**Comment by claudiulodro** [comment]
That change was from [this pull request](https://github.com/woocommerce/woocommerce/pull/20520) and is scheduled to be included in 3.5.0. The merging system is fine. :)

**Comment by dtugend** [comment]
Thanks and sorry.
